content-test/selectors/siteMetadataSelectors.test.js

```diff
@@ -82,10 +82,15 @@ describe("siteMetadataSelectors", () => {
 
       assert.ok(result.parsedUrl);
     });
-    it("should set label prop", () => {
+    it("should set label prop without eTLD set", () => {
       let result = selectSiteProperties(validSpotlightSite);
 
-      assert.ok(result.label);
+      assert.equal(result.label, "cnn.com");
```

Comment:
So there's `result.parsedUrl.hostname` but that actually returns "www.cnn.com". Any reason this shouldn't be a plain string comparison for the test?